site/includes/skiplinks.hbs

```diff
@@ -1,15 +1,17 @@
-<ul id="wb-tphp">
-	<li class="wb-slc">
-		<a class="wb-sl" href="#wb-cont">{{{i18n "tmpl-skip-cont"}}}</a>
-	</li>
-{{#isnt footer "false"}}
-	<li class="wb-slc">
-		<a class="wb-sl" href="#wb-info">{{{i18n "tmpl-skip-about-government"}}}</a>
-	</li>
-{{/isnt}}
-{{#if page.secondarymenu}}
-	<li class="wb-slc visible-md visible-lg">
-		<a class="wb-sl" href="#wb-sec">{{{i18n "tmpl-skip-section"}}}</a>
-	</li>
-{{/if}}
-</ul>
+<nav>
```

Comment:
@shawnthompson , @bsouster 

What do you think if we add ```aria-label="Skip Link and Accessibility Options"``` to the ```<nav>``` element.

That is what the WAI initiative home page do.

Reply:
I missed this comment. I'm up for that or even a invisible heading like we do with all our other sections.

Reply:
@shawnthompson Did you want to do it in CDTS too?

Reply:
fyi - That aria-label is not implemented in GCWeb.